table/table.cpp

```diff
@@ -231,10 +261,10 @@ string Table::get_type(string &column) {
             type_str = "STRING";
             break;
     }
-
     return type_str;
 }
 
+// Outputs Table row by row, entry by entry
```

Comment:
Since this is a `void` function printing stuff out and not actually returning anything, it would be more clear to describe this as "Prints" instead of "Outputs" since "Outputs" sometimes implies the return value.

Apply throughout.